agent/internal/options/options.go

```diff
@@ -81,10 +80,6 @@ type Options struct {
 	// master config.
 	AgentReconnectBackoff int `json:"agent_reconnect_backoff"`
 
-	ContainerRuntime   string             `json:"container_runtime"`
```

Comment:
what about ```image_root```?

https://docs.determined.ai/latest/reference/deploy/agent-config-reference.html#image-root

docs this have any affect on Docker?

Reply:
I am not very clear of your question here.

If you mean that removing Singularity/Podman/Apptainer as container_runtime for AgentRM might affect with setting those image cache in [image_root](https://docs.determined.ai/latest/reference/deploy/agent-config-reference.html#image-root) parameter - Looks unlikely as we know we have these options as is for when deploying Slurm though Launcher or any non-agent way shown [here](https://docs.determined.ai/latest/setup-cluster/slurm/singularity.html#provide-a-container-image-cache)

Or if you mean that as AgentRM won't be supporting Singularity/Podman/Apptainer, hence it would have to make corrections in [here](https://docs.determined.ai/latest/setup-cluster/slurm/singularity.html#singularity-image-cache). I do not think docker will be affected even in this case.

Kindly correct if my understanding is not right

Reply:
I think there is two ```image_root``` config options.

One lives in master config with ```resource_manager.agent_root``` while the other is in agent config as  ```image_root```

For the agent config option, I think that Docker does not use the ```image_root``` option given it was added for other runtimes. So I think now that we have removed the other runtimes I think we can also remove ```image_root``` if Docker does not use it.